tools/csv-merger/csv-merger.go

```diff
@@ -129,8 +129,15 @@ func main() {
 
 	for _, image := range strings.Split(*relatedImagesList, ",") {
 		if image != "" {
-			names := strings.Split(strings.Split(image, "@")[0], "/")
-			name := names[len(names)-1]
+			name := ""
+			if strings.Contains(image, "|") {
+				image_s := strings.Split(image, "|")
```

Comment:
If a user would like to pass `--related-images-list` parameter to `csv-merger` tool (via `build-manifest.sh`?) with both image URL **and** a label, he/she should use '|' as the seperator between url and label.
I think it should be pointed out that the seperator is this character, for example in line 77(?)

Reply:
Done